Security.md

```diff
@@ -0,0 +1,123 @@
+# Security
+
+The security model of WebAssembly has two important goals: (1) protect *users*
+from buggy or malicious modules, and (2) provide *developers* with useful
+primitives and mitigations for developing safe applications, within the
+constraints of (1).
+
+## Users
+
+Each WebAssembly module executes within a sandboxed environment separated from
+the host runtime using fault isolation techniques. This implies:
+  * Applications execute independently (unless
+  [dynamic linking](DynamicLinking.md) is used), and can't escape the sandbox
```

Comment:
Applications execute independently even if dynamic linking is used. It's just that an application consists of more than one module. Probably the reference to dynamic linking isn't necessary here.